src/templates/resource.js

```diff
@@ -15,12 +15,12 @@ export const ResourceTemplate = ({ title, body, date }) => (
     date={date} />
 )
 
-const Resource = ({ data }) => {
+const Resource = ({ data, pageContext }) => {
   const { markdownRemark: post } = data
   const title = post.frontmatter.heading ?
     post.frontmatter.heading : post.frontmatter.title
 
-  return <App title={title}>
+  return <App title={title} locale={pageContext.locale}>
```

Comment:
Ahh, *now* we're feeling like Haskell. How far down can we thread this one piece of state that everything in the system needs? :p

Reply:
Yeah, passing `locale` down as a prop through several children isn't the best. Added it to React context to get around this in upcoming commit https://github.com/keep-network/tbtc-website/pull/158/commits/b398d4ed051f02bad07f06533104da44f0e0a6fa